gradle.properties

```diff
@@ -2,3 +2,6 @@ version=1.0-SNAPSHOT
 platformlib-process.version=0.1.4
 gradle-node-plugin.version=3.5.0
 octopus-components-registry-service.version=2.0.27
+junit-jupiter-api.version=5.10.2
+junit-jupiter-engine.version=5.10.2
```

Comment:
what is reason to have two version if api and engine are updated togethed?

Reply:
The two version parameters offer flexibility, allowing for the use of different versions of the API and engine when needed, such as for compatibility reasons or when only one component is updated.